admin/index.html

```diff
@@ -162,10 +162,10 @@
             }
         });
 
-        if (obj.devices.hg1 === 'Auto') {
-            obj.devices['hg0'] = 'Auto';
+        if (obj.devices.bm1 === 'Auto' || obj.devices.bm2 === 'Auto' || obj.devices.bm3 === 'Auto' || obj.devices.bm4 === 'Auto') {
```

Comment:
Does hg1 no more actual?

Reply:
"Corrected issue with hg0 (BWL-1-S) being activated when hg1 (TOB / CGB-2 / MGK-2) is active, made it its own group (picture should be updated)"
hg0 = BWL-1-S - Please refer to the new code lines 383-399
hg0 was in the old code being added to IOBroker even though it didn't exist.